djautotask/sync.py

```diff
@@ -7,6 +7,7 @@
 from django.utils import timezone
 
 from djautotask import api, models
+from djautotask.models import PROJECT_STATUS_COMPLETE
```

Comment:
You are already importing models, so you can just access this with `models.PROJECT_STATUS_COMPLETE`

Reply:
Actually, this would probably be better included in the `ProjectStatus` class. Then, as a class attribute, you could access it like `ProjectStatus.COMPLETE`, which looks a lot cleaner to me.

Reply:
Nice idea. Fixed